server/views/challenges/showZiplineOrBasejump.jade

```diff
@@ -8,11 +8,10 @@ block content
                     | &nbsp;
                     i.ion-checkmark-circled.text-primary(title="Completed")
             hr
-            ol
+            ul
                 for step, index in description
                     .row.checklist-element(id="#{dashedName + index}")
-                        .col-xs-3.col-sm-1.col-md-2.padded-ionic-icon.text-center
-                            input(type='checkbox' class='challenge-list-checkbox')
+                        .col-xs-3.col-sm-1.col-md-2
```

Comment:
@sanskar-modi **pic[2]** looks the best. Thanks! It may not be good practice, however, to have an empty `div` just for padding?